helm/dagster/schema/schema/charts/dagster/values.py

```diff
@@ -22,6 +22,7 @@ class DagsterHelmValues(BaseModel):
     computeLogManager: subschema.ComputeLogManager
     scheduler: subschema.Scheduler
     runLauncher: subschema.RunLauncher
+    pythonLogs: Optional[subschema.PythonLogs]
```

Comment:
Are there any sensible defaults that we can have for this? Making this optional hurts discoverability. Maybe have an `enabled: bool` field in `PythonLogs` if we want a toggle?  

If so, we should put those defaults in the `values.yaml` file. For Helm folks, the `values.yaml` is the entrypoint for configuration of their deployment. If they don't see `pythonLogs` as a top level key there, it would be pretty hard for them to discover this.